### DISCUSSION TITLE
Poll: Which os version of custom client would you expect mostly? Pro users only

### DISCUSSION THREAD

**Discussion by rustdesk** [rustdesk/rustdesk-server-pro#217] [Polls]
# We do not have enough resource, so have to develop one by one. Pro users only please

**Comment by DMSnz** [comment]
@rustdesk 
How hard to get an Android builder for the immediate need, just needs our server key configured?
We have CK65 mobile scanners I am desperately trying to remote today, but typing such a key in will be a nightmare, any way to preconfigure APK with server host + key but really key would be enough.

**Reply by DMSnz** [reply]
@rustdesk 
![20240408_104123](https://github.com/rustdesk/rustdesk-server-pro/assets/139725889/ec3e6cf6-ec98-42db-bde5-728a183ccd89)

Found another point to raise regarding Android client...
Our CK65 devices have small low resolution screens and today I tried to manually set one up with api server, key etc only to get stuck on the SCAM warning prompt, the agree/disagree buttons that I see on my phone, are not visible on the CK65.
Give further push to wanting a custom Android client builder, that can hopefully disable the scam warning for one thing alongside other requests like deploying with host/key.
We use a MDM so can install self created apps "Enterprise" apps as its called.
I would be willing to pay a bounty to push this development forward any clues how to do that and I will jump on it.

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/pull/6846 https://github.com/rustdesk/rustdesk/pull/6850 fixed this, it does not work for you?

**Reply by rustdesk** [reply]
Android is harder than others. We won't do a only ＂our server key configured＂solution.

**Reply by DMSnz** [reply]
I downloaded the latest Android APK this morning, assuming fix was client side?
Did not work it seems

Can we bounty something so I can get a workable solution in regards to your last comment, server keys etc. Happy to contribute to get this over the line as its not scalable for our solution right now, where I hate to say it, TeamViewer works fine.

**Reply by rustdesk** [reply]
> https://github.com/rustdesk/rustdesk/pull/6846 https://github.com/rustdesk/rustdesk/pull/6850 fixed this, it does not work for you?

@sahilyeole could you check this?

**Reply by rustdesk** [reply]
Yes, a bounty can push this.

Does `"Enterprise" apps` mean this https://github.com/rustdesk/rustdesk/discussions/6610?

**Reply by beelchester** [reply]
>https://github.com/rustdesk/rustdesk/pull/6846 https://github.com/rustdesk/rustdesk/pull/6850 fixed this, it does not work for you?
>@sahilyeole could you check this?

This is Nexus S emulator with a similar resolution and size as CK65.
Latest nightly:

https://github.com/rustdesk/rustdesk-server-pro/assets/73148455/e46c29e1-3365-4d62-8c5f-942d1d5a650f

1.2.3-2 has this problem. You can try out installing the nightly version.

<img width="389" alt="Screenshot 2024-04-08 at 8 51 52 AM" src="https://github.com/rustdesk/rustdesk-server-pro/assets/73148455/42d3987b-d587-4104-b430-64e6f7866860">

**Reply by DMSnz** [reply]
@rustdesk Enterprise Apps does mean that yes, but I use ManageEngine (Zoho) and do NOT have such an issue, can upload it fine and distribute to devices no issue, we also use Managed Google Play on these devices, further suggesting no issue with ManageEngine at least.

**Reply by DMSnz** [reply]
@sahilyeole Thank you, confirming that nightly build 1.2.4 is working and on our small device screens we can scroll down to see the Accept/Decline options.

**Comment by rustdesk** [comment]
MacOS is ready, https://twitter.com/rustdesk/status/1777180459429986787

**Comment by rustdesk** [comment]
MSI and Android ready now, https://twitter.com/rustdesk/status/1782250852574769663

**Comment by rustdesk** [comment]
Linux is ready now, https://twitter.com/rustdesk/status/1788905463678951787
